examples/deferred_processing/README.md

```diff
@@ -0,0 +1,340 @@
+# Deferred Processing and Transactions
+
+## Introduction
+[Deferred processing](https://docs.pachyderm.com/latest/how-tos/deferred_processing/) 
+is a Pachyderm technique for controlling when data gets processed.
+Transactions are Pachyderm feature for assuring data from different repos get processed together.
+
+This is an example which uses a simple DAG, 
+based on our opencv example, 
+to illustrate using deferred processing along with transactions.
+
+
+
+## Pipelines
+
+![Example DAG](./example_dag.png)
+
+The DAG shown is a simple elaboration on the opencv example,
+with pipeline and repo names chosen to avoid collisions with that example 
+if installed in the same cluster.
+
+The functionality is, of course, different.
+The `edges_dp` pipeline will do edge detection on images committed to `images_dp_1`.
+`montage_dp` will create a montage out of images committed to `images_dp_2` and images in the master branch of `edges_dp`. 
+This is so it's easy to verify the files being processed
+by visually inspecting the montage.
+
+The most significant change from opencv in this example is the pipeline spec for `edges_dp`,
+which has the `output_branch` attribute set to `dev`.
+
+```json hl_lines="11"
+{
+    "pipeline": {
+        "name": "edges_dp"
+    },
+    "input": {
+        "pfs": {
+            "glob": "/*",
+            "repo": "images_dp_1"
+        }
+    },
+    "output_branch": "dev",
+    "transform": {
+        "cmd": [ "python3", "/edges.py" ],
+        "image": "pachyderm/opencv"
+    }
+}
+```
+
+This means that this pipeline, 
+instead of putting its output in the `master` branch,
+will put it in a branch called `dev`.
+
+Since `montage_dp` is subscribed to the master branch of `edges_dp`,
```

Comment:
```suggestion
Since `montage_dp` is subscribed to the master branch of `edges_dp`, jobs will not trigger when the edges pipeline outputs files to the `dev` branch. Instead, to trigger a montage job, 
```